### PR TITLE
Adding pt-br translations (pt-br.json)

### PR DESCRIPTION
**Pull request by jaideraf**
This PR addresses or contains:
- Adding pt-br translations (not in translatewiki.net)

This PR includes:
- [ ] Tests (unit/integration)
- [x] CI build passed

### CODE REVIEW

**Comment by mwjames** [comment]
@jaideraf  Your time and effort is much appreciated. @kghbln I'm leaving you in charge for review and merge.

File: i18n/extra/pt-br.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
+{
+    "@metadata": {
+        "authors": []
```
Comment (kghbln) [inline i18n/extra/pt-br.json:3]: Don't be timid. You can add your user name here. :)

File: i18n/extra/pt-br.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    },
+    "fallback_language": "en",
```
Comment (kghbln) [inline i18n/extra/pt-br.json:5]: I am not an expert on Portuguese however it may be better to define "pt" as the fall-back language rather than English.

**Comment by kghbln** [comment]
@jaideraf Cool, thanks for the pull. I just completed my review.

**Comment by jaideraf** [comment]
Done! Thank you both. I do my best to make them consistent. With these translations, SMW in pt-br is 100% translated: https://translatewiki.net/w/i.php?title=Special:MessageGroupStats&language=pt-br&group=mwgithub-semanticmediawiki#sortable:3=desc :100:

**Comment by kghbln** [comment]
> Done! Thank you both.

Cool. You are the best and not just I know it. :) Will merge once the tests pass.

**Comment by mwjames** [comment]
Should be added to the release notes to avoid any surprises since it has an impact on namespace and predefined property representations.

**Comment by kghbln** [comment]
Indeed, forgot to tag the pull. Done.

**Pull request merged**